packages/swingset-runner/demo/vatStore2/thingHolder.js

```diff
@@ -42,17 +44,25 @@ function build(name) {
   const thingMaker = makeKind(makeThingInstance);
   let nextThingNumber = 0;
 
-  const myThings = makeVirtualScalarWeakMap('thing'); // thing -> inquiry count
+  let myThings;
+
+  function ensureCollection() {
+    if (!myThings) {
+      myThings = makeScalarBigWeakMapStore('things');
+    }
+  }
 
   return Far('root', {
     async introduce(other) {
       const otherName = await E(other).getName();
       const thing = thingMaker(`thing-${nextThingNumber}`, other, otherName);
```

Comment:
This demonstrates why I don't like this naming convention. This is the making function that's exported by the defining module to clients who should be able to make these. They shouldn't need to know that it is virtual, which the non-standard naming convention doesn't hint at anyway.
```suggestion
      const thing = makeThing(`thing-${nextThingNumber}`, other, otherName);
```

Reply:
I think I had it the way I did was because 
```
const makeThing = makeKind(makeThingInstance);
```
felt like too much `makeFoo` in one line.  But it's not a deeply held position and if you'd rather shift the naming pattern I'm happy to do that.